README.md

```diff
@@ -1078,5 +1066,4 @@ Use your browser(CTL-F) to search by country code.
 - [Melissa Data](http://www.melissadata.com/lookups/index.htm)
 
 #### Other Interesting
-- [iknowwhatyoudownload](https://iknowwhatyoudownload.com/en/)
-- [Amazon Wish List Search](www.amazon.com/gp/registry/search)
+- [Findsubdomains](https://findsubdomains.com/)
```

Comment:
Can't accept this, I already have it in another repo https://github.com/Ph055a/Domains_OSINT/blob/master/README.md